__tests__/index.ts

```diff
@@ -0,0 +1,16 @@
+import { from, until } from '../src/index'
+
+describe('until', () => {
+	test('is an object that provides breakpoint media queries', () => {
+		expect(until.small).toBe('@media all and (max-width: 29.9375em)')
+	})
+})
+
+describe('from', () => {
+	test('is a weird old beast', () => {
+		expect(from.small.toString()).toBe('@media all and (min-width: 30em)')
```

Comment:
this bit worries me, in that unable so far to prove this actually works. it _should_ do though, right?

Reply:
Is this any more explicit...
```js
expect(`${from.small}`).toBe('@media all and (min-width: 30em)')
```
It's closer to the actual syntax

Reply:
i'll merge it so it's a bit easier to see everything in one go

Reply:
you end up with this:

```
    Expected: "@media all and (min-width: 30em)"
    Received: {"toString": [Function toString], "until": {"large": etc...}
```

Reply:
so question is is this a quirk of the testing env or correct?